docs/community/styleguide/formatting.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Formatting style guide
+description: This style guide explains using correct formats when writing documentation.
+weight: 
+---
+## Documentation formatting
+
+### Notes and warning blocks
+
+Notes and warning blocks are used to draw attention to important information. Use the following markdown features when necessary:
+
+- Use a clear and concise heading to introduce the note or warning.
+- Use short paragraphs or bullet points to convey the information.
+- Keep the language simple and direct.
+- Use an `>` in markdown to indicate the nature of the note or warning. 
+- Use the following Use the following syntax to apply a style. Currently our documenation supports **Remember** `<Remember>`:
+  * Surround the text with an opening and closing tag.
+  * Use the following syntax to apply a style:
+  ` <Remember> 
+  No need to add a prefix (Remember); the tag automatically provides one
+  </Remember>`
+  
+  The output: 
+  <Remember> 
+  No need to add a prefix (Remember); the tag automatically provides one
+  </Remember>
```

Comment:
This entire section is confusing, and I can't understand it. 😄✌🏽 What are you documenting here? It is not clear that you're explaining that we have a  React component for creating a remember card. 

Also, why do you have the `(remember)` in the middle of that sentence in line 20? What do you mean by "no need to add a prefix"? What prefix are you talking about? 👀 

Please rewrite this section and think about all the pieces you are trying to educate the reader on ✌🏽

Reply:
even after adding tags some people might think they need to write "Remember" inside of tags to make it work. That's what I wanted to address